javascript/src/stories/components/ChannelMessages/MessageListView.stories.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-
+import React from "react"
```

Comment:
Is there a reason there is no semicolon on this line?

Reply:
I must have copied the import from somewhere else, I'll quickly add one